### DISCUSSION TITLE
shorts auto skip

### DISCUSSION THREAD

**Discussion by UnknownSkeletalRemains** [arichornlover/uYouEnhanced#16] [Q&A]
is there a tweak that automatically skips shorts when watch it? or i need to activate some flags in a/b? 
btw thx for fixing app crashing on shorts 😊

**Comment by arichornlover** [comment]
> is there a tweak that automatically skips shorts when watch it? or i need to activate some flags in a/b?

so far no, but will be looking around to see if that’s possible.

> btw thx for fixing app crashing on shorts 😊

wdym, I haven’t fixed it yet? Using v18.05.2 breaks and crashes shorts

**Comment by UnknownSkeletalRemains** [comment]
oops, I confused with Cercube Plus Extra

**Reply by arichornlover** [reply]
Yeah ik CercubePlusExtra doesn’t have the crashing issue unlike uYou’s.
